src/widgets/FileManagerInputWidget.php

```diff
@@ -71,11 +121,11 @@ public function init()
                 'url'            => $this->to('search', null),
                 'dataType'       => 'json',
                 'delay'          => 220,
-                'data'           => new JsExpression('searchData'),
+                'data'           => new JsExpression($this->jsGetSearchDataFuncName),
```

Comment:
Maybe you should add an if(is_string) (or similar) around that because the JSExpression returns an [empty string ('') for an empty value (null)](https://github.com/yiisoft/yii2/blob/master/framework/web/JsExpression.php#L46) and that could lead to problems.

Reply:
I'm a big fan of checks before using variables anywhere, but in this case a protected variable is used which is set in the int() of the widget (see line 108). So I don't see a scenario here where the variable could be set “incorrectly” or have i missed something?